src/resources/js/formdirectives.js

```diff
@@ -2047,6 +2057,7 @@ zaa.directive("zaaCheckboxArray", function () {
                         '<label for="{{id}}">{{label}}</label>' +
                     '</div>' +
                     '<div class="form-side">' +
+                        '<button ng-click="toggleAll()" type="button" class="ckeckbox-list-toggle-button btn btn-info btn-icon"><i class="material-icons">done_all</i></button>' +
```

Comment:
Rename `checkbox-list-toggle-button` to `zaa-checkbox-array-toggle-all`, we already have a `zaa-checkbox-array.scss` file which defines some `zaaCheckboxArray` spcific things.